tests/base/src/test/java/org/wildfly/security/http/digest/DigestAuthenticationMechanismTest.java

```diff
@@ -60,6 +60,17 @@ public static void removePasswordProvider() {
         Security.removeProvider(provider.getName());
     }
 
+    public void testDigest(String []authorization,HttpServerAuthenticationMechanism mechanism) throws Exception{
```

Comment:
@keshav-redhat just a minor, since both of these methods can obtain any authorization header, even different than digest header, we can name it based on its contents. Maybe eg. `testRequest` or `testAuthorizationRequest`?